examples/nlp/bert_pretraining.py

```diff
@@ -54,17 +57,22 @@
                                    files_to_copy=[__file__],
                                    add_time_to_log_dir=True)
 
-special_tokens = ['PAD', '[UNK]', '[CLS]', '[SEP]', '[MASK]']
+special_tokens = ['[PAD]', '[UNK]', '[CLS]', '[SEP]', '[MASK]']
 data_desc = BERTPretrainingDataDesc(args.dataset_name,
                                     args.data_dir,
                                     args.vocab_size,
                                     args.sample_size,
                                     special_tokens,
                                     'train.txt')
 
-tokenizer = nemo_nlp.SentencePieceTokenizer(
-    model_path=data_desc.tokenizer_model)
-tokenizer.add_special_tokens(special_tokens)
+if args.tokenizer == "sentence-piece":
+    tokenizer = nemo_nlp.SentencePieceTokenizer(
+        model_path=data_desc.tokenizer_model)
+    tokenizer.add_special_tokens(special_tokens)
+else:
```

Comment:
Please make this explicit (case specifically for "nemo-bert") and have the else case raise an error.
We want to make sure that if the user mistypes `sentence-piec` or something that we won't silently fall through to a different default than they expect.

Reply:
Thanks! Revised the code accordingly.